### ISSUE TITLE
Consider supporting png images for modelmaker object detector

### ISSUE THREAD

**Issue by Malthehave** [tensorflow/tensorflow#54391]
**System information**
- TensorFlow version (you are using): 2.7.0

**Describe the feature and the current behavior/state.**
Currently the `object_detector` class only supports jpg images. This can be a problem, especially if you have already created label annotations for the images in another image format. I suggest that we consider adding support for png files as well.
**Will this change the current api? How?**
No
**Who will benefit with this feature?**
Everyone using png images instead of jpg for training data.
**Any Other info.**
As far as I can see then there is a check for file format here on line 133 of the `dict_to_tf_example` function: https://github.com/tensorflow/examples/blob/035fb73d5ff8b74958c9e3f83f44fc20dfc39119/tensorflow_examples/lite/model_maker/third_party/efficientdet/dataset/create_pascal_tfrecord.py#L133

**Comment by sushreebarsa** [comment]
@Malthehave 
Can you please elaborate about your feature and please specify the use cases for this feature? Thanks!

**Comment by Malthehave** [comment]
Hi @sushreebarsa, sure thing. 
For clarification I'm talking about the `tflite_model_maker` library.
As of right now you will get an error if you try to train the network with .png images, by calling the [ObjectDetector.create](https://www.tensorflow.org/lite/api_docs/python/tflite_model_maker/object_detector/ObjectDetector#create) method. 
A basic use case, is that you have already created your annotations in PASCAL VOC format, pointing to your PNG images.

**Comment by Fatichti** [comment]
Hi

Do you make an improvment to use png file with the tflite_model_maker library ?

**Comment by pkgoogle** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFLite team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the TFLite space.

Thanks.
